scripts/Component.py

```diff
@@ -98,6 +98,7 @@ def getFilters(self, mapping, config):
                     "IceSSL/configuration",
                     "IceDiscovery/simple",
                     "IceGrid/simple",
+                    "IceUtil/stacktrace",
```

Comment:
Enable this test for static builds. Otherwise, it's hard to see if we have an issue with backtraces and static builds!